src/ConcordiumNetSdk/Wallets/Json/Key.cs

```diff
@@ -0,0 +1,15 @@
+namespace ConcordiumNetSdk.Wallets.Json;
+
+/// <summary>
+/// Represents (a subset of) a JSON object indexed by the
+/// <c>key</c> field of the browser and genesis wallet
+/// export JSON formats.
+///
+/// Such can be parsed into an instance of this class using
+/// <see cref="Newtonsoft.Json.JsonConvert"/>.
+/// </summary>
+public record Key
```

Comment:
Does this need to be public?

Reply:
Do you believe it should be `internal`?

Reply:
As a general rule the smaller the API surface is the better since it is easier to find what you need.

And if these keys are only internal then I would make them as such.